// @storyblok/react/dist/rsc.mjs
import { default as t, default as r } from "./bridge-loader.mjs";
import { getComponent as a, getCustomFallbackComponent as i, getEnableFallbackComponent as s, useStoryblokApi as b, setComponents as d, storyblokInit as k, useStoryblokApi as n } from "./index2.mjs";
import { default as g } from "./story.mjs";
import { default as m } from "./storyblok-component.mjs";
import { RichTextResolver as f, RichTextSchema as u, apiPlugin as x, loadStoryblokBridge as c, registerStoryblokBridge as B, renderRichText as C, storyblokEditable as h, registerStoryblokBridge as R } from "./storyblok-js.mjs";
export {
  t as BridgeLoader,
  f as RichTextResolver,
  u as RichTextSchema,
  r as StoryblokBridgeLoader,
  m as StoryblokComponent,
  g as StoryblokStory,
  x as apiPlugin,
  a as getComponent,
  i as getCustomFallbackComponent,
  s as getEnableFallbackComponent,
  b as getStoryblokApi,
  c as loadStoryblokBridge,
  B as registerStoryblokBridge,
  C as renderRichText,
  d as setComponents,
  h as storyblokEditable,
  k as storyblokInit,
  n as useStoryblokApi,
  R as useStoryblokBridge
};
